src/main/java/com/spartronics4915/frc2019/subsystems/Drive.java

```diff
@@ -340,6 +344,16 @@ public void outputTelemetry()
         {
             mCSVWriter.write();
         }
+
+        dashboardPutString("leftDemand", mPeriodicIO.left_demand + "");
+        dashboardPutString("rightDemand", mPeriodicIO.right_demand + "");
+        if (mDriveControlState == DriveControlState.VELOCITY || mDriveControlState == DriveControlState.PATH_FOLLOWING)
+        {
+            dashboardPutString("leftSpeedTarget", ticksPer100msToInchesPerSecond(mPeriodicIO.left_demand) + "");
+            dashboardPutString("rightSpeedTarget", ticksPer100msToInchesPerSecond(mPeriodicIO.right_demand) + "");
+            dashboardPutString("leftSpeedFeedforward", mPeriodicIO.left_feedforward + "");
```

Comment:
minor nit:  leftSpeedFeedforward -> leftFeedforward (its not a speed, its a voltage)